External/Plugins/ResultsPanel/PluginMain.cs

```diff
@@ -271,17 +251,16 @@ public void CreatePluginPanel()
         {
             this.pluginUI = new PluginUI(this);
             this.pluginUI.Text = TextHelper.GetString("Title.PluginPanel");
-            this.pluginPanel = PluginBase.MainForm.CreateDockablePanel(this.pluginUI, this.pluginGuid, this.pluginImage, DockState.DockBottomAutoHide);
-
-            this.panelHelper = new ResultsPanelHelper(this);
+            this.pluginUI.ParentPanel = PluginBase.MainForm.CreateDockablePanel(this.pluginUI, this.pluginGuid, this.pluginImage, DockState.DockBottomAutoHide);
+            ResultsPanelHelper.Initialize(this, this.pluginUI);
         }
         
         /// <summary>
         /// Opens the plugin panel if closed
         /// </summary>
         public void OpenPanel(Object sender, EventArgs e)
         {
-            this.pluginPanel.Show();
+            ResultsPanelHelper.ActiveUI.ParentPanel.Show();
```

Comment:
This is when the user uses the shortcut "ViewMenu.ShowResults", or clicks View > Results Panel.
Would it be better to change it back to `this.pluginUI.ParentPanel.Show()`?